src/App.jsx

```diff
@@ -2,14 +2,14 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(setCurrent => setCurrent + 1);
```

Comment:
You should use better naming for variables. `setCurrent` here is not a function, so maybe you should figure out a better name.🙃